packages/yourloops/lib/language.ts

```diff
@@ -117,5 +118,24 @@ function t(s: string, p?: TOptions | string): string {
   return i18n.t(`yourloops|${s}`, p);
 }
 
-export { init, t };
+const getCurrentLocaleName = (i18n: i18next): string => {
+  const shortLocale = i18n.language.split("-")[0] as "en" | "de" | "es" | "fr" | "it" | "nl";
+
+  return locales.resources[shortLocale]?.name;
+};
+
+const getLocaleShortname = (locale: string): string => {
+  let shortName = "";
+  _.forEach(locales.resources, ({ name }, key) => {
```

Comment:
What !!!
a for each !!
😉

Reply:
This one iterates over the keys of an object